bpf/http_sock.c

```diff
@@ -121,7 +121,6 @@ int BPF_KPROBE(kprobe_tcp_rcv_established, struct sock *sk, struct sk_buff *skb)
         // If the source port for a client call is lower, we'll get this wrong.
         // TODO: Need to fix this. 
         pid_info.orig_dport = pid_info.p_conn.conn.s_port,
-        task_tid(&pid_info.c_tid);
```

Comment:
This isn't required anymore. I figured out a better way to track what I need for cleaning up the parent trace information. Essentially, now I store the namespaced threadID and the extra runtime ID on the HTTP request. It's the most accurate information to use on cleanup, since we use those to create the parent request trace.